matrix/tasks/reset.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+
+
+async def reset(context, rule, task, event=None):
```

Comment:
I am fine with this, but, this may end up being destroy the old model and create a new one as we talked about once.